howso/feature_residuals.amlg

```diff
@@ -621,6 +625,118 @@
 						(unzip feature_residuals_lists numeric_continuous_indices)
 					)
 				)
+			;smape = |predicted - actual| / ((|actual| + |predicted|) / 2) * 100
+			smape_map
+				(zip
+					numeric_continuous_features
+					;iterate over continuous features, where (current_value) is a list of react tuples for each feature
+					(map
+						(lambda
+							(if (size (current_value))
+								(*
+									(/
+										;sum of the absolute percent errors
+										(apply "+"
+											(map
+												(lambda
+													(let
+														(assoc
+															smape_numerator
+																(abs (-
+																	(get (current_value 1) predicted_offset)
+																	(get (current_value 1) actual_offset)
+																))
+															smape_denominator
+																(/ (+
```

Comment:
all closing and opening parents should line up and be on same corresponding lines so please move the (+ to the next line.